core/templates/local_vector.h

```diff
@@ -40,13 +40,39 @@
 
 // If tight, it grows strictly as much as needed.
 // Otherwise, it grows exponentially (the default and what you want in most cases).
+// force_trivial is used to avoid T's default value on resize, for improved performance.
+// This requires T to be trivially destructible.
 template <typename T, typename U = uint32_t, bool force_trivial = false, bool tight = false>
 class LocalVector {
 private:
 	U count = 0;
 	U capacity = 0;
 	T *data = nullptr;
 
+	template <bool p_init>
+	void _resize(U p_size) {
+		if (p_size < count) {
+			if constexpr (!std::is_trivially_destructible_v<T>) {
+				for (U i = p_size; i < count; i++) {
+					data[i].~T();
+				}
+			}
+			count = p_size;
+		} else if (p_size > count) {
+			if (unlikely(p_size > capacity)) {
+				capacity = tight ? p_size : nearest_power_of_2_templated(p_size);
```

Comment:
Shouldn't this have used a 1.5x factor instead of 2x?

Reply:
Ohh, good catch! I must have missed the change on a rebase.

I think it's too late to re-introduce 1.5x growth factor now for 4.5, but I'll make a PR for early 4.6.